tools/logger/convert.c

```diff
@@ -624,7 +624,13 @@ static int read_entry_from_ldc_file(struct ldc_entry *entry, uint32_t log_entry_
 	entry->params = NULL;
 
 	/* set file position to beginning of processed entry */
-	fseek(global_config->ldc_fd, entry_offset, SEEK_SET);
+	ret = fseek(global_config->ldc_fd, entry_offset, SEEK_SET);
+	if (ret) {
+		log_err("Failed to seek to entry header for offset 0x%x in dictionary.\n",
+			entry_offset);
+		ret = -1;
```

Comment:
line 631 does not need, ret either 0 or -1.

Reply:
fread function returns
> Number of objects read successfully
so in case on an error ret will be 0, not -1.